### PR TITLE
Created available materials

### PR DESCRIPTION
**Pull request by iPR0D1G1**
Renamed available-materials to available versions.
Created aailabe-materials and finished design.

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/tunnel/summary/summary.component.html [hunk 1]
```diff
@@ -102,12 +102,18 @@ <h3>Technical Specifications</h3>
...
+      <movie-summary-available-versions [movie]="form" [link]="('available-versions' | getPath)">
+      </movie-summary-available-versions>
+    </section>
 
+    <!-- AVAILABLE MATERIALS -->
     <section id="available-materials">
       <h3>Available Materials</h3>
-      <movie-summary-available-materials [movie]="form" [link]="('available-materials' | getPath)">
-      </movie-summary-available-materials>
+      <movie-summary-available-versions [movie]="form" [link]="('available-materials' | getPath)">
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/tunnel/summary/summary.component.html:115]: you've got twice `movie-summary-available-versions` I think you mean `movie-summary-available-materials`

File: libs/movie/src/lib/movie/form/shell/shell.component.ts [hunk 2]
```diff
@@ -55,10 +55,21 @@ function getSteps(status: ProductionStatus, appSteps: TunnelStep[] = []): Tunnel
       path: 'technical-spec',
       label: 'Technical Specification'
     }, {
+      path: 'available-versions',
+      label: 'Available Versions',
+      shouldHide: isStatus(status, ['development'])
+    }]
+  }, {
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/shell/shell.component.ts:62]: This part should not be in the lib. It's specific to the catalog application. Add it in `apps\catalog\catalog\src\app\dashboard\tunnel\movie-tunnel.routes.ts`. You can take example in `apps\financiers\financiers\src\app\dashboard\tunnel\routes.ts`

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/movie/src/lib/movie/+state/movie.firestore.ts [hunk 3]
```diff
@@ -84,7 +84,10 @@ export interface MovieBase<D> {
   synopsis: string, //! required
   title: Title, //! required
   orgIds: string[] //! required
-  campaignStarted: D
+  campaignStarted: D,
+
+  //CATALOG specific
+  availableMaterials?:StorageFile,
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/+state/movie.firestore.ts:90]: Please put it as :
```typescript
delivery?: {
  file: StorageFile;
}
```
And update the rest of the code accordingly

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

**Review by iPR0D1G1 — COMMENTED** [review]
reviewed

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**